app/Http/Controllers/CampaignsController.php

```diff
@@ -107,13 +106,15 @@ public function showCampaign($id)
 
         return view('pages.campaign_single')
             ->with('state', [
-                'signups' => $signups,
+                'signups' => $signups->toArray()['data'],
```

Comment:
Oh right, got it! If you want, you could switch to `$signups->items()` to grab the items directly.